pubspec.yaml

```diff
@@ -63,8 +63,9 @@ dependencies:
   dartesr:
     git:
       url: https://github.com/digital-scarcity/dartesr.git
-    
-dev_dependencies:
+  qr_mobile_vision: ^1.0.2
```

Comment:
What is this dependency?

Reply:
qr_mobile_vision is alternative package to scan QR ( so barcode_scanner can be removed from dependencies )
It allows to handle QR callback without closing scanner ( so we can continue scanning in case of error )